setup.cfg

```diff
@@ -13,8 +13,10 @@ addopts = -v --doctest-modules --ignore=larray/core/ufuncs.py --ignore=larray/ip
 # E201: whitespace after '['
 # E202: whitespace before ']'
 # E241: multiple spaces after ','
+# E303: too many blank lines
 # E712: comparison to True should be 'if cond is True:' or 'if cond:'
 pep8ignore =
-     #E127
+     E127
```

Comment:
I personally would rather avoid a global ignore on all files for such an important check, even if it means a few cases of less than optimal code